src/BcPrebidVast.js

```diff
@@ -73,16 +77,34 @@ function dispatchPrebidDoneEvent() {
 	document.dispatchEvent(event);
 }
 
-// the function loads prebid.js and does header bidding if needed
+// This function will be invoked immediately when plugin script is loaded in the page or by brightcove player.
```

Comment:
consider using JSdocs for documenting function signatures. Take a look at prebid.js for some examples https://github.com/prebid/Prebid.js/blob/master/src/prebid.js#L86